### PR TITLE
Generate brokerHost in Broker-CI pipeline

### PR DESCRIPTION
**Pull request by fadidurah**
Small PR to add BrokerHost and OldBrokerHost APKs to fix some broken test cases in this release pipeline: https://identitydivision.visualstudio.com/Engineering/_build?definitionId=1490&_a=summary

### CODE REVIEW

File: azure-pipelines/ui-automation/broker-test.yml [hunk 1]
```diff
@@ -163,6 +170,9 @@ stages:
     - script: mkdir cpauth
       displayName: 'make cpauth dir'
       workingDirectory: '$(Build.ArtifactStagingDirectory)'
+    - script: mkdir brokerhost
```
Comment (shahzaibj) [inline azure-pipelines/ui-automation/broker-test.yml:173]: Why is this needed?

Comment (shahzaibj) [reply azure-pipelines/ui-automation/broker-test.yml:173]: Did we need to do this in other pipelines too?

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:173]: adding BrokerHost APKs to cpauth dir didn't make much sense, and I didn't want to change the name of that directory since I don't know if other pipeline might be pulling from it

**Review by shahzaibj — APPROVED** [review]
Left one comment, otherwise LGTM

File: azure-pipelines/ui-automation/broker-test.yml (hunk 1 above)
Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:173]: not that I know of, this is a new directory being added to this pipeline, don't think it would affect any other pipelines

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:173]: Actually I guess the name of this directory doesn't matter, since if other pipelines were pulling from anything, they'd be pulling from the published directory holding the artifacts (`brokerapks`). So I can change that dir's name without worry.

**Review by somalaya — APPROVED** [review]
